docs/source/membership/membership.md

```diff
@@ -1,341 +1,194 @@
+you've seen how a Public Key Infrastructure (PKI) can provide verifiable identities through a chain of trust. How is that chain of trust used by the blockchain network? Recall from the credit card scenario in the Identity topic that the Certificate Authority is like a card
+provider — it dispenses many different types of verifiable identities. An MSP, on the other hand, determines which credit card providers are accepted at the store. In this way, the MSP turns an identity (the credit card) into a role (the ability to buy things at the store).
+
+This ability to turn verifiable identities into roles is fundamental to the way Fabric networks function, since it allows organizations, nodes, and channels the ability establish MSPs that determine who is allowed to do what at the organization, node, and channel level.
+
+![MSP1a](./membership.msp.diagram.png)
+
+*Identities are similar to your credit cards that are used to prove you can pay. The MSP is similar to the list of accepted credit cards.*
+
+Consider a consortium of banks that operate a blockchain network. Each bank operates peer and ordering nodes, and the peers endorse transactions submitted to the network. However, each bank would also have departments and account holders. The account holders would belong to each organization, but would not run nodes on the network. They would only interact with the system from their mobile or web application. So how does the network recognize and differentiate these identities? A CA was used to create the identities, but like the card example, those identities cant just be issued, they need to be recognized by the network. MSP's are used to define the organizations that are trusted by the network members. MSPs are also the mechanism that provide members with a set of roles and permissions within the network.
+
+Finally, consider if you want to join an _existing_ network, you need a way to turn your identity into something that is recognized by the network. The MSP is the mechanism that enables you to participate on a permissioned blockchain network. To transact on a Fabric network a member needs to:
+
+- Have an identity issued by a CA that is trusted by the network.
+- Become a member of an _organization_ that is recognized and approved by the network members. The MSP is how the identity is linked to the membership of an organization. Membership is achieved by adding the member's public key (also known as certificate, signing cert, or signcert) to the organization’s MSP.
+
+## What is an MSP?
+
+Despite its name, the Membership Service Provider does not actually provide anything. Rather, the implementation of the MSP requirement is a folder that contains a list of related identities and is used to store users's certificates or in the case of channel configuration, a list of organizations. Whereas Certificate Authorities generate the certificates that represent identities, the MSP contains a list of permissioned identities.
+
```

Comment:
Rather, MSPs are a set of folders that are added to the configuration of network and used to define an organization both inwardly (organizations decide who its admins are) and outwardly (allowing other organizations to validate that entities have the authority to do what they are attempting to do.